### PR TITLE
Resize widget based on height as well as width

### PR DESCRIPTION
**Pull request by spasiu**
@dannytranlx @lemieux @Mario54 @liyefei737 

PREVIOUSLY:

screen shot 2016-06-22 at 12 32 08 pm

AND NOW:

screen shot 2016-06-22 at 12 30 41 pm

screen shot 2016-06-22 at 12 30 56 pm

screen shot 2016-06-22 at 12 31 05 pm

Width based sizing remains the same as before.

### CODE REVIEW

**Comment by lemieux** [comment]
pls test on all devices

**Comment by chloepouprom** [comment]
And don't forget to modify this file, since the animations have media query breakpoints too: https://github.com/smooch/smooch-js/blob/integration/src/stylesheets/animations.less

**Comment by mspensieri** [comment]
👍

**Comment by liyefei737** [comment]
👍

**Comment by dannytranlx** [comment]
👍  Beauty commits 😁

File: src/stylesheets/animations.less [hunk 1]
```diff
@@ -170,11 +210,24 @@
...
+    }
+
+    @media (max-height: @screen-sm-ht-max) {
         bottom: @widget-close-bottom-sm;
     }
+
+    @media (max-width: @screen-sm-min) {
+        bottom: @widget-close-bottom-fs;
+    }
 }
```
Comment (chloepouprom) [inline src/stylesheets/animations.less]: When `max-height: @screen-sm-ht-max and min-width: @screen-lg-min` the widget is hidden

Comment (spasiu) [inline src/stylesheets/animations.less]: I think something is wrong with your local environment.

**Pull request merged**